src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.mm

```diff
@@ -43,19 +43,25 @@ static BOOL isRunningTests(void)
     return (environment[@"XCTestConfigurationFilePath"] != nil);
 }
 
-CHIP_ERROR CHIPOperationalCredentialsDelegate::init(CHIPPersistentStorageDelegateBridge * storage)
+CHIP_ERROR CHIPOperationalCredentialsDelegate::init(CHIPPersistentStorageDelegateBridge * storage, ChipP256KeypairPtr nocSigner)
 {
     if (storage == nil) {
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+    CHIP_ERROR err = CHIP_NO_ERROR;
     mStorage = storage;
 
-    CHIP_ERROR err = LoadKeysFromKeyChain();
+    if (!nocSigner || nocSigner->Initialize() != CHIP_NO_ERROR) {
```

Comment:
`Initialize()` in `CHIPCryptoPAL` implementations creates a new keypair. I think, it's better to expect that provide `nocSigner` is already initialized by the caller (either by creating a new keypair, or retrieving one from the storage).

If we call `Initialize()` every time, it'll end up creating a new signer keypair on each run.